invenio_app_rdm/config.py

```diff
@@ -484,27 +484,37 @@ def files_rest_permission_factory(obj, action):
 APP_RDM_RECORD_EXPORTERS = {
     "json": {
         "name": _("JSON"),
-        "serializer": ("flask_resources.serializers:JSONSerializer")
+        "serializer": ("flask_resources.serializers:JSONSerializer"),
+        "content-type": "application/json",
+        "filename": "{id}.json"
```

Comment:
I understand those were placed here to simplify the task (don't need to touch invenio-rdm-records). Maybe in the future, putting the `content-type` and `filename` in the serializer directly would nicely group coupled responsibility, reduce configuration and keep it dry. It makes sense that the `JSONSerializer` has a content-type of `application/json` and so on. Repeating it is tedious :) . I made an issue for it: https://github.com/inveniosoftware/invenio-rdm-records/issues/817